### DISCUSSION TITLE
flet build crashed

### DISCUSSION THREAD

**Discussion by jk-2022** [flet-dev/flet#3668] [General]
j'utilise la bibliothèque aiohttp et asyncio pour construire mon application flet=0.23.2, 
mais j'obtiens des erreurs dans la constructions des wheels aiohttp`:

VERBOSE: Using cached six-1.16.0-py2.py3-none-any.whl (11 kB)
(     ●) Packaging Python app ⏳... VERBOSE: Building wheels for collected packages: aiohttp
(    ● ) Packaging Python app ⏳... VERBOSE: Building wheel for aiohttp (pyproject.toml): started
(   ●  ) Packaging Python app ⏳... VERBOSE: Building wheel for aiohttp (pyproject.toml): finished with status 'error'
VERBOSE: Failed to build aiohttp

**Comment by ndonkoHenri** [comment]
For which platform are you trying to package your app?

**Reply by jk-2022** [reply]
android

**Reply by jk-2022** [reply]
finalement, j'ai opter pour httpx
